.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,42 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master ]
+    tags: [ v* ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+    paths-ignore: [ '**/*.md' ]
+  schedule:
+    - cron: '00 14 * * *' # Every morning at 7:00am PDT
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript' ]
```

Comment:
I'm double checking that this isn't supposed to be 'typescript'.

Reply:
I guess it could be either lol https://github.com/github/codeql-action/blob/03f029c2a15a75961a2724371dcbbab4bd33ce47/src/languages.ts#L16

Reply:
Testing...